server/src/test/java/org/apache/iotdb/db/query/reader/resourceRelated/SeqResourceReaderTest.java

```diff
@@ -94,7 +97,7 @@ protected void insertData() throws IOException, QueryProcessException {
     }
     storageGroupProcessor.waitForAllCurrentTsFileProcessorsClosed();
 
-    assert storageGroupProcessor.getWorkSequenceTsFileProcessor() == null;
+    assert storageGroupProcessor.getWorkSequenceTsFileProcessor().isEmpty();
```

Comment:
Use assertions in JUnit, this won't wake effect without -ea option.

Reply:
fixed~